packages/mini-browser/src/node/mini-browser-endpoint.ts

```diff
@@ -180,7 +180,7 @@ export class MiniBrowserEndpoint implements BackendApplicationContribution, Mini
             const mimeType = lookup(FileUri.fsPath(stat.uri));
             if (!mimeType) {
                 this.logger.warn(`Cannot handle unexpected resource. URI: ${statWithContent.stat.uri}.`);
-                return response.send();
+                return response.send(content);
```

Comment:
Please
```js
if (!mimeType) {
    this.logger.warn(`Cannot handle unexpected resource. URI: ${statWithContent.stat.uri}.`);
} else {
    response.contentType(mimeType);
}
return response.send(content);
```

Reply:
According to [`mime-type`](https://www.npmjs.com/package/mime-types):
> No fallbacks. Instead of naively returning the first available type, mime-types simply returns false, so do var type = mime.lookup('unrecognized') || 'application/octet-stream'.

I have updated my snippet above 👆

Reply:
@kittaakos Do we need to log warning? We don't have it for webviews and so far nobody complained.

Reply:
> Do we need to log warning? We don't have it for webviews and so far nobody complained.

I do not have a strong preference; if you feel we need it, please proceed. Otherwise, we can merge the PR.

Reply:
It is to avoid unnecessary noise. I'm fine with merging as is.